### PR TITLE
fix prerequisites

### PR DESCRIPTION
**Pull request by ThibaultMiam**

### CODE REVIEW

**Comment by ThibaultMiam** [comment]
* **#21** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/18?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#17** <a href="https://app.graphite.dev/github/pr/miamtech/mealz-documentation/17?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ThibaultMiam and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/android/features/_shared/Prerequisites.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### Packages
-To have the Catalog functioning properly, you must install our packages `miamCore` & `androidSDK`.
+To have the Catalog functioning properly, you must install our packages `mealz-core` & `mealz-android`.
```
Comment (antonin-francois) [inline docs/android/features/_shared/Prerequisites.md:2]: peut-être faire la même côté iOS ?

File: docs/android/features/_shared/Prerequisites.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 [Here are the instructions](/docs/android/overview/installation)
 
 ### Initialize Mealz
-Also, you must initialize `Mealz` in your MainActivity.
-[Here are the instructions](/docs/android/overview/initialisation)
\ No newline at end of file
+Also, you must initialize `Mealz` in your MainActivity. Here are the instructions
+[if you are a provider](/docs/android/overview/providerInit) or here [if you are a supplier](/docs/android/overview/supplierInit), if you don't know witch configuration you need [check here](/docs/android/overview/initialisation#initialisation-types)
```
Comment (diarmuidmcg) [inline docs/android/features/_shared/Prerequisites.md:7]: witch = 🧙
```suggestion
[if you are a provider](/docs/android/overview/providerInit) or here [if you are a supplier](/docs/android/overview/supplierInit), if you don't know which configuration you need [check here](/docs/android/overview/initialisation#initialisation-types)
```

File: docs/android/features/_shared/Prerequisites.md (hunk 1 above)
Comment (ThibaultMiam) [reply docs/android/features/_shared/Prerequisites.md:2]: oui tu as raison il y a une autre tâche pour ça

**Pull request merged**